src/SwedbankPay/Core/Library/Methods/Checkout.php

```diff
@@ -10,6 +10,48 @@
 
 trait Checkout
 {
+	/**
+	 * Check Checkout API Credentials.
+	 *
+	 * @return bool Returns true if ok
+	 * @throws Exception
+	 */
+	public function checkCheckoutApiCredentials()
+	{
+		/**
+		 * Perform a POST request to create a payment or paymentorder with invalid data that is known to fail with
+		 * 400 Bad Request.
+		 * This is to ensure that no payments or payment orders are actually created if the request succeeds.
+		 * If the request fails with 401, something is wrong with the credentials.
+		 * If the request fails with 403 something is wrong with the contract.
```

Comment:
Can `401` and `403` throw different exceptions with more concrete information about what may be wrong and how to fix it?